### DISCUSSION TITLE
[node] v14/fs.d.ts is missing `throwIfNoEntry` in StatOptions etc.

### DISCUSSION THREAD

**Discussion by rrthomas** [DefinitelyTyped/DefinitelyTyped#55786] [Issues with a @types package]
See e.g. https://nodejs.org/docs/latest-v14.x/api/fs.html#fs_fs_statsync_path_options

The `throwIfNoEntry` flag is implemented in Node 14, but the types bindings for v14 do not include it; meanwhile, it was implemented in the types bindings for v15 and later.

[a comment by an automated account is not shown]

**Comment by SimonSchick** [comment]
Yh generally backports happen rarely for node typings (I really dislike doing them), but I might look into it later, super busy this week.

**Reply by rrthomas** [reply]
Thanks, it would be super if you did find time!

**Comment by rrthomas** [comment]
As a workaround in the mean time, the following is adapted from code in the TypeScript runtime:

```
const stats: fs.Stats = (fs as any).statSync(path, {throwIfNoEntry: false})
```
